tests/IdentifyServiceProviderTest.php

```diff
@@ -2,8 +2,9 @@
 
 namespace Unicodeveloper\Identify\Test;
 
-use Mockery\MockInterface;
+use \Mockery as m;
```

Comment:
Please, could you remove the backslash @perfectmak ? I don't think that's necessary